pages/_app.tsx

```diff
@@ -45,7 +45,7 @@ const MainLayout = ({ Component, pageProps }) => {
           site_name: data.title,
           images: [
             {
-              url: 'https://tinacms.org/img/tina-twitter-share.png',
+              url: './public/img/tina-og.png',
```

Comment:
does this need the `./public` ?

Reply:
i would imagine so

Reply:
https://www.opengraph.xyz/url/https%3A%2F%2Ftina-jx24qfqr2-tinacms.vercel.app%2F

Seems like somethings wrong

Reply:
yes - was using path dir not url